### PR TITLE
Readme corrections

### PR DESCRIPTION
**Pull request by BenAlderfer**
Some wording changes, removed duplicate gcal section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,16 +65,6 @@ You only need to do this if you are working on mentoring-related endpoints.
 * Head back to [Google Developer Console](https://console.developers.google.com/apis/library). Go to the same project you created before. Go to Libraries and search for Google Calendar API. Enable that API.
 * Next go to Credentials Tab. Click Create Credentials > API Key. You can store this key in `keys/google.json` under `web.api_key` or set the ENV variable `GOOGLE_API_KEY`.
 
-### Google Calendar
```
Comment (billdybas) [inline README.md]: Why remove this? IIRC, the section above is for OAuth and this one is for setting up Google Calendar.

**Comment by BenAlderfer** [comment]
See above, there's a duplicate of the gcal stuff right above it

File: README.md (hunk 1 above)
Comment (BenAlderfer) [reply README.md]: See above, there's a duplicate of the gcal stuff right above it

File: README.md [hunk 2]
```diff
@@ -27,6 +27,8 @@ The API uses Google OAuth for authentication. You'll need to create a Project us
 #### Configuring Google OAuth
 The following steps walk you through configuring Google OAuth, with the assumption you are using it for development in conjunction with the [OneRepoToRuleThemAll](https://github.com/rit-sse/OneRepoToRuleThemAll) project. **If you are using another client, you will have to modify Step 6 to conform with that client's configuration.**
 
+Note - OAuth seems to silently fail on Windows, even running with WSL 1. It is suggested you use a unix environment (macOS or a Linux distro)
```
Comment (akath20) [inline README.md]: Do we have an issue tracking this we can link to? Additionally, if you have Windows 10 Pro with Hyper-V, did it silently fail in Docker? If not, did you setup the environment outside of Docker? Just to assist those who try to pick it up next know what you tried.

Comment (BenAlderfer) [reply README.md]: I didn't bother with docker or Hyper-V. Just setup wsl and tried running it natively like in a linux environment

**Pull request merged**